libshaderc_util/src/compiler.cc

```diff
@@ -230,6 +230,14 @@ std::tuple<bool, std::vector<uint32_t>, size_t> Compiler::Compile(
   // Note the call to GlslangToSpv also populates compilation_output_data.
   glslang::GlslangToSpv(*program.getIntermediate(used_shader_stage), spirv);
 
+  // Set the tool field (the top 16-bits) in the generator word to
+  // 'Shaderc over Glslang'.
```

Comment:
Point taken. 
The vendor field is "Google" and the tool field is "Shaderc over Glslang".
The SPIRV-Tools disassembler jams the two together.  But that's a detail we don't need to address here.